lib/puppet_x/templates/init_ps.ps1.erb

```diff
@@ -391,6 +395,12 @@ function Reset-ProcessPowerShellVariables
     [Void]$ps.AddScript($ourFunctions)
     $ps.Invoke()
 
+    if ($WorkingDirectory -ne '') {
+      if (-not (Test-Path -Path $WorkingDirectory)) { Throw "Working directory `"$WorkingDirectory`" does not exist" }
+      $ps.Runspace.SessionStateProxy.Path.SetLocation($WorkingDirectory)
+    } else {
+      $ps.Runspace.SessionStateProxy.Path.SetLocation($global:DefaultWorkingDirectory)    
```

Comment:
I'm not seeing where the extra spaces are.  The `if (-not ...` is in the right spot